### DISCUSSION TITLE
how to hire developers from github ?

### DISCUSSION THREAD

**Discussion by djmohsen** [community/community#20218] [Other Feature Feedback, Questions, & Ideas]
hello,
**i would like to pay & hire developers wich have participated in project (jigsaw-outline) in github, they are about 31 programmers ,** 
_i need a fork from project with customization for me_

how to talk and have co operation with them ?
https://github.com/Jigsaw-Code/outline-client

**Comment by D7EAD** [comment]
If they have contact information on their profiles, you could always contact them that way!
